docs/ldml/tr35-dates.md

```diff
@@ -79,7 +79,7 @@ The LDML specification is divided into the following parts:
   * Table: [timeZoneNames Elements Used for Fallback](#timeZoneNames_Elements_Used_for_Fallback)
   * [Metazone Names](#Metazone_Names)
 * [Supplemental Time Zone Data](#Supplemental_Time_Zone_Data)
-  * [Metazones](#Metazones)
+  * [Metazones](#metazones)
```

Comment:
@macchiati I guess the downside here is that `(#Metazones)` no longer works _from within the .md file_

Reply:
It seems like we should have the "canonical" anchor match whatever github wants, to enable the markdown links, but then add other forms for compatibility.